src/main/java/net/dries007/tfc/ConfigTFC.java

```diff
@@ -359,6 +359,21 @@ public static void onConfigChangedEvent(ConfigChangedEvent.OnConfigChangedEvent
         @Config.RangeInt(min = 0, max = 2)
         @Config.LangKey("config." + MOD_ID + ".general.inventoryCraftingTableMode")
         public int inventoryCraftingTableMode = 0;
+
+        @Config.Comment("How much can a small vessel hold?")
```

Comment:
```suggestion
        @Config.Comment("How much metal (units / mB) can a small vessel hold?")
```
And the same for others, including the units here would be better